Bottom: 7b527f711768166a0eaa950c02710df6fdd564f1
Top:    52f2408fae2e8631c5a797277d5bcb245a0c4511
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 11:56:14 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/fsl/dspeak01_fabric.c b/sound/soc/fsl/dspeak01_fabric.c
index c6c7890..bd35a14 100644
--- a/sound/soc/fsl/dspeak01_fabric.c
+++ b/sound/soc/fsl/dspeak01_fabric.c
@@ -113,7 +113,7 @@ static int __devinit dspeak01_fabric_probe(struct of_device *op,
 	if (!fabric.clock)
 		return -ENODEV;
 
-	of_snd_soc_register_fabric("DSPEAK01", &dspeak01_fabric_ops, NULL);
+	of_snd_soc_register_machine("DSPEAK01", &dspeak01_fabric_ops);
 	return 0;
 }
